doc/rados/troubleshooting/troubleshooting-osd.rst

```diff
@@ -245,16 +303,17 @@ Or::
 	osd.4 is backfill full at 91%
 	osd.2 is near full at 87%
 
-The best way to deal with a full cluster is to add new ``ceph-osds``, allowing
-the cluster to redistribute data to the newly available storage.
+The best way to deal with a full cluster is to add capacity via new OSDs, enabling
+the cluster to redistribute data to newly available storage.
 
-If you cannot start an OSD because it is full, you may delete some data by deleting
-some placement group directories in the full OSD.
+If you cannot start a legacy Filestore OSD because it is full, you may reclaim
+some space deleting a few placement group directories in the full OSD.
 
 .. important:: If you choose to delete a placement group directory on a full OSD,
    **DO NOT** delete the same placement group directory on another full OSD, or
-   **YOU MAY LOSE DATA**. You **MUST** maintain at least one copy of your data on
-   at least one OSD.
+   **YOU WILL LOSE DATA**. You **MUST** maintain at least one copy of your data on
+   at least one OSD.  This is a rare and extreme intervention not to be undertaken
```

Comment:
s/rare and extreme intervention not/rare and extreme intervention, and is not/

I think this makes the sentence a little clearer.

Reply:
I think it's more dramatic as written, but don't object.

Reply:
On repeated readings, I see your point.